src/app/page.tsx

```diff
@@ -48,7 +49,23 @@ export default function Home() {
             <OurClientButton key={item.description} {...item} />
           ))}
         </div>
-      </div>
+      </Section>
+      <Section className="flex flex-col px-12 gap-8">
+        <span className="text-gray-600 italic underline">Testimonials</span>
+        <div className="grid grid-cols-2 gap-8 px-24">
```

Comment:
👍  on mobile though this should have only 1 col right?

Reply:
I'm developing at `macOS first` model 😆 In the end I'll come back to all this sections to make it mobile responsive, but yes, it will probably have just 1 column